Dockerfile

```diff
@@ -9,15 +9,17 @@
 #
 ###
 
-FROM alpine:3.2
+FROM alpine:latest
 MAINTAINER Kingsquare <docker@kingsquare.nl>
+MAINTAINER bubak4 <mar@centrum.cz>
 
 ENV SSH_AUTH_SOCK /ssh-agent
 
 ####
-# Install the SSH-client
-RUN apk add --update openssh-client && rm -rf /var/cache/apk/*
+# Install the autossh
+RUN apk add --update autossh && rm -rf /var/cache/apk/*
 
 VOLUME ["/ssh-agent"]
-EXPOSE 2222 
-ENTRYPOINT ["/usr/bin/ssh", "-T", "-N", "-o", "StrictHostKeyChecking=false", "-o", "ServerAliveInterval=180", "-L"]
+# for ambassador mode
+EXPOSE 2222
+ENTRYPOINT ["/usr/bin/autossh", "-M", "0", "-T", "-N", "-oStrictHostKeyChecking=no", "-L"]
```

Comment:
I use autossh reliably without the ServerAliveInterval, but I did some search on the Internet and it is recommended to be used, so be it :-)